### PR TITLE
Add sketches to the intro for acrtive bird detail pages

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

- go to various bird detail pages

## Show me
_Provide screenshots/animated gifs/videos if necessary._

Screen Shot 2022-09-29 at 12 33 56 PM

# REMEMBER: Attach this PR to the Trello card

### CODE REVIEW

**Comment by mirisuzanne** [comment]
[Add individual author headshot sketches to the author detail pages](https://trello.com/c/ExesTvtQ/177-add-individual-author-headshot-sketches-to-the-author-detail-pages)

File: src/filters/birds.cjs [hunk 1]
```diff
@@ -48,7 +48,7 @@ params:
     note: The name of the bird (as used in `author` settings)
 */
 const authorPage = (collection, bird) =>
-  withData(collection, 'data.bird', bird)[0];
+  withData(collection, 'data.bird', bird, true)[0];
```
Comment (jgerigmeyer) [inline src/filters/birds.cjs:51]: @mirisuzanne This fixed a bug where `david` was returning truthy for bird `davidh`. We may want to check to see if there are other places this is an issue.

**Review by jgerigmeyer — APPROVED** [review]

**Review by SondraE — APPROVED** [review]
Looks really nice. 
Those headshots really draw attention to the blurbs. Makes me wonder if we want to standardize the type of information included. Right now some people list hobbies. Others list language expertise. Others list skills. 
Also this blurb could be the author info that goes at the bottom of an article. If so, then I'd feel even more strongly about standardizing the type of info provided for each person.

**Comment by mirisuzanne** [comment]
@SondraE I suppose you could open a new card and make a proposal on how to standardize?

File: src/filters/birds.cjs (hunk 1 above)
Comment (jgerigmeyer) [reply src/filters/birds.cjs:51]: Fixed in https://github.com/oddbird/oddleventy/pull/245

**Pull request merged**